ehsm_kms_service/delete_secret_thread.js

```diff
@@ -0,0 +1,123 @@
+const {
+    base64_decode
+} = require('./function')
+
+// Set and initialize a interval
+var interval_time_instance
+
+// Set a queue to record the waiting time
+var delete_task_queue = {}
+
+/**
+ * remove a timer from exist timer by key
+ * @param {Object} DB : database control
+ * @param {Object} key : key form exist timer
+ * @returns
+ */
+const remove_delete_task = (secretName, appid, DB) => {
+    try {
+        //Delete thread when thread exists
+        if (delete_task_queue[secretName + '_' + appid] != undefined ) {
+            clearTimeout(delete_task_queue[secretName + '_' + appid].timer)
+        }
+        delete delete_task_queue[secretName + '_' + appid];
+        // An insurance mechanism to ensure that thread interval_time_instance is active
```

Comment:
does it actually need? if it have issue, how to you consider the other timer? they also should have the same issue..

Reply:
ok, I will delete it